coralproject/talk
# [CORL-2432]: mod comment card top bar styles updates

## What does this PR do?

These changes include a few small style changes that make it so that the feature button in the moderate comment card will no longer overlap with other content when there is a lot of content on the top line, such as a long username. I also updated a few other small style things that I came across, such as the `timestamp` needed its classname and should be using a smaller font size (I removed its other styles as they are redundant with the `Timestamp` component). I also removed the `inReplyTo` classname, as it didn't have any styles associated with it.

## What changes to the GraphQL/Database Schema does this PR introduce?

none

## Does this PR introduce any new environment variables or feature flags? 

no

## If any indexes were added, were they added to `INDEXES.md`?

n/a

## How do I test this PR?

You can go to a moderate comment card. View a comment by a user with a long username. You can also make them always pre-moderated or banned or suspended so that they also get a tag, so there are more things included on that line of the moderate comment card. Make your browser different sizes to see how it looks and confirm that the feature button no longer overlaps with other content at smaller sizes. Check out other moderate comment cards with different usernames, content, etc., to see that they also look fine.
 
## How do we deploy this PR?
